src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -113,7 +114,9 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
         </Button>
       </>,
       version,
-      scaStatus,
+      <>
```

Comment:
It's a good question; I was having trouble verifying but I think there must be something in Patternfly setting the key for the rows (this gets passed into the table's rows props), otherwise it would be yelling at us.

Reply:
It looks like there is a rule about whether to check shorthand fragments for keys and it defaults to false: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-key.md#checkfragmentshorthand-default-false

Reply:
aha ok I just enabled that rule and added keys to that fragment and the fragment above it. 

I still think that Patternfly probably has some mechanism of adding keys to the rows, otherwise React should be yelling about it in the console. But good to add that rule in in general.

Reply:
Is the fragment still needed?  Couldn't the keys just go directly on the inner components?

Reply:
Yeah I had thought the same but it throws a strange error if you take out the fragments around the Patternfly components. Must be something with how the Patternfly row is implemented, is my best guess.

<img width="774" alt="Screen Shot 2021-05-26 at 9 45 46 PM" src="https://user-images.githubusercontent.com/4838984/119722346-801c0f00-be6c-11eb-8a18-90386215f538.png">
<img width="1457" alt="Screen Shot 2021-05-26 at 9 45 55 PM" src="https://user-images.githubusercontent.com/4838984/119722357-827e6900-be6c-11eb-8c2f-55a3ae00c4c9.png">
<img width="896" alt="Screen Shot 2021-05-26 at 9 50 47 PM" src="https://user-images.githubusercontent.com/4838984/119722367-84e0c300-be6c-11eb-9ab7-805131196d59.png">

Reply:
Obviously you should have used an array instead of an array.

I guess I'll discontinue worrying about this until the next time it comes up.